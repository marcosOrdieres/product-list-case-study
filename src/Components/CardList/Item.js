import styled from 'styled-components';

const Item = styled.div`
  align-items: center;
  position: relative;
  border-bottom: 1px solid 5px;
  padding: 5px;

  &:first-child {
    border-top-left-radius: 1px;
    border-top-right-radius: 1px;
  }

  &:last-child {
    border-bottom-left-radius: 1px;
    border-bottom-right-radius: 1px;
  }
`;

Item.displayName = 'CardList.Item';

export default Item;
