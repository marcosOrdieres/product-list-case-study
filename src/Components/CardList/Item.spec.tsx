import { render } from '@testing-library/react';
import React, { Fragment } from 'react';

import Item from './Item';

describe('Card List Items', () => {
  it('should render matching default styles', () => {
    const { asFragment } = render(
      <Fragment>
        <Item>1</Item>
        <Item>2</Item>
        <Item>3</Item>
      </Fragment>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
