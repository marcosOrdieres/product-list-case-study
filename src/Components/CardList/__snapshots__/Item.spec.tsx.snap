// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card List Items should render matching default styles 1`] = `
<DocumentFragment>
  <div
    class="sc-bdfBQB bujJht"
  >
    1
  </div>
  <div
    class="sc-bdfBQB bujJht"
  >
    2
  </div>
  <div
    class="sc-bdfBQB bujJht"
  >
    3
  </div>
</DocumentFragment>
`;
